src/_base.scss

```diff
@@ -5,29 +5,29 @@
 
 /// Base quote styles
 @mixin oQuote {
+	@include oTypographyPadding($top: 10, $bottom: 10);
+
+	// sass-lint:disable no-vendor-prefixes
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
```

Comment:
What is the reason behind using a different font smoothing? Could we have this as a comment?